packages/cli/src/templates/nextjs-[pages].tsx

```diff
@@ -0,0 +1,62 @@
+import { useRouter } from 'next/router';
+import { BuilderComponent, builder, useIsPreviewing } from '@builder.io/react';
+import DefaultErrorPage from 'next/error';
+import Head from 'next/head';
+
+// Replace with your Public API Key
+builder.init('<<<YOUR_API_KEY>>>');
+
+export async function getStaticProps({ params }) {
+  // Fetch the builder content
+  const page = await builder
+    .get('<<<MODEL_NAME>>>', {
+      userAttributes: {
+        urlPath: '/' + (params?.page?.join('/') || ''),
+      },
+    })
+    .toPromise();
+
+  return {
+    props: {
+      page: page || null,
+      revalidate: 5,
+    },
+  };
+}
+
+export async function getStaticPaths() {
+  // Get a list of all pages in builder
+  const pages = await builder.getAll('<<<MODEL_NAME>>>', {
+    // We only need the URL field
+    fields: 'data.url',
+    options: { noTargeting: true },
```

Comment:
PS: I believe you need a `limit: 0` here, the default limit is 30

https://github.com/BuilderIO/builder/blob/d7e477991aa5fdc2cee7a5b9ce033dba6fe0e8dc/packages/core/src/builder.class.ts#L2580